File: 백준/Bronze/24768. Left Beehind/Left Beehind.cc
#include <iostream>
using namespace std;
int main(void) {
    int x,y;
    while(1) {
        cin>>x>>y;
        if(x==0 && y==0) break;
        if(x+y==13)
            cout<<"Never speak again.\n";
        else {
            if(x>y) cout<<"To the convention.\n";
            else if(x<y) cout<<"Left beehind.\n";
            else cout<<"Undecided.\n";
        }
    }
    return 0;
}

File: 백준/Bronze/24768. Left Beehind/Left Beehind.py
import sys
input=lambda:sys.stdin.readline().rstrip()
while 1:
    x,y=map(int,input().split())
    if x==0 and y==0: break
    if x+y!=13:
        if x>y:
            print("To the convention.")
        elif x<y:
            print("Left beehind.")
        else:
            print("Undecided.")
    else:
        print("Never speak again.")